haskell/src/SwiftNav/SBP/Acquisition.hs

```diff
@@ -0,0 +1,26 @@
+module SwiftNav.SBP.Acquisition where
```

Comment:
Do you need to export the messages here as well, and also instantiate a Data.HashMap?